app/src/components/styled/generic/StyledText.ts

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
```

Comment:
same above. Maybe rename combine it with headings and rename both to "typeography.ts"
 in components